src/components/fields/DataSelector.js

```diff
@@ -119,7 +124,6 @@ UnconnectedDataSelector.propTypes = {
   fullValue: PropTypes.any,
   updatePlot: PropTypes.func,
   container: PropTypes.object,
-  placeholder: PropTypes.string,
```

Comment:
but don't we still want to have that capability of adding our custom string..
i guess we'll readjust when that becomes needed

Reply:
yep.